test/Makefile.am

```diff
@@ -86,3 +88,6 @@ regen-abi:
 	@echo "    abidw $(ABIDW_FLAGS) /path/to/libz.so > libz.abi"
 
 build-TESTS: $(check_PROGRAMS) $(check_SCRIPTS)
+
+clean-local:
+	-rm -f test_gzfile*.gz
```

Comment:
This rule is not needed since the test is already deleting the `test_gzfile*.gz` files.

Reply:
It's just to guarantee make clean cleans everything, if the test do not finish the files will stay there.